### PR TITLE
You can no longer drop the clown shoes once you put them on your feet.

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
tweak: The clown shoes can no longer be taken off your feet.
/:cl:

Once a clown, always a clown. I dislike seeing clowns taking a metaphorical dump on the nature of clowns by taking off their shoes and taking mutadone, as the "strategy" is as of more recently. When you pick a clown, you willingly acknowledge the clown's upsides and downsides. People shouldn't pick clown only for the free slips.

### CODE REVIEW

**Comment by ghost** [comment]
yes

**Comment by KayeArray** [comment]
no

**Comment by ghost** [comment]
also sinful but not as sinful

**Comment by ghost** [comment]
basically clowns are cursed from roundstart.

-1

**Comment by ghost** [comment]
YES

if you're going CLOWN you need to COMMIT

+100000000

**Comment by dreamcrab** [comment]
https://en.wikipedia.org/wiki/Clown_(film)

reminds me of this movie

**Comment by ghost** [comment]
new strategy as clown: cutting legs off then getting robotic legs

**Comment by GuyonBroadway** [comment]
--Edit-- Never-mind I see you accounted for that.

File: hippiestation/code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -24,3 +24,7 @@
 		log_admin("[key_name(user)] dealt brain damage to [key_name(M)] with the Extra annoying bike horn")
 
 #undef HORN_BRAIN_DAMAGE
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/clown_items.dm]: undefs need to go at the very bottom of a file please place your code ABOVE any undefs

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

**Review by GuyonBroadway — APPROVED** [review]

File: hippiestation/code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/obj/item/clothing/shoes/clown_shoes/equipped/equipped(mob/user, slot)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/clothing/shoes/miscellaneous.dm:1]: Now this means that if you cut a clowns legs off and picked his shoes up they would get stuck to your hand.

Might wanna make it so they get undone on an unequip.

**Comment by tortellinitony** [comment]
stupid

**Comment by ghost** [comment]
Potential change: give clowns clown slowdown by nature,  or even slower, and clownshoes give them normal clownshoe slowdown. If they wanna go fast they gotta get nuka cola,  or ephedrine.  Traitor clowns are excused.  Oooor....  Traitor clowns get 25 tc to compensate slower movespeed

**Comment by ghost** [comment]
forcing a particular playstyle is kinda shit though
even if these "clowns" are only in name related.

if clowns take their shoes off then the IC ire they get is justified, coding against it is bad though

**Comment by ghost** [comment]
how about giving clowns with clownshoes on a special asskick attack.  It only knocks people back   baseball bat style.  Hey,  mime has free forcewall so why not

**Comment by Arzion5** [comment]
I didn't actually take into account traitor clowns.

...shit.

**Pull request closed without merging**